examples/HM-WDS40-TH-I/HM-WDS40-TH-I.ino

```diff
@@ -88,7 +89,10 @@ typedef Sht31<> SensorType; // I2C
 #include <sensors/Si7021.h>
 typedef Si7021 SensorType; // I2C
 #endif
-
+#ifdef SENSOR_AM2301
+#include <sensors/Dht.h>
+typedef Dht<3,AM2301> SensorType; // <DataPin, Type>
```

Comment:
I think it would be good to use a define for the data pin too.

Reply:
I added a define for the DataPin for the DHT22 and AM2301 sensors.